packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayoutNew/CoinmarketLayoutNavigation/CoinmarketLayoutNavigation.tsx

```diff
@@ -1,17 +1,31 @@
 import styled from 'styled-components';
-import { useDevice } from 'src/hooks/suite';
+import { useDevice, useSelector } from 'src/hooks/suite';
 import { FirmwareType } from '@trezor/connect';
 import CoinmarketLayoutNavigationItem from './CoinmarketLayoutNavigationItem';
+import { Divider } from '@trezor/components';
+import { spacings } from '@trezor/theme';
+import { useMemo } from 'react';
+import regional from 'src/constants/wallet/coinmarket/regional';
 
 const List = styled.div`
     display: flex;
     flex-wrap: wrap;
     align-items: center;
 `;
 
+const SeparatorWrapper = styled.div`
+    height: 42px;
+`;
+
 const CoinmarketLayoutNavigation = () => {
     const { device } = useDevice();
     const isBitcoinOnly = device?.firmwareType === FirmwareType.BitcoinOnly;
+    const country = useSelector(
+        state =>
+            state.wallet.coinmarket.buy.buyInfo?.buyInfo?.country ??
+            state.wallet.coinmarket.sell.sellInfo?.sellList?.country,
+    );
+    const isInEEA = useMemo(() => Boolean(country && regional.isInEEA(country)), [country]);
```

Comment:
Maybe we should we hide it if tor is active

Reply:
Is this solved?

Reply:
yes